tests/metrics/test_disease_observer.py

```diff
@@ -218,3 +227,58 @@ def test_observation_correctness(base_config, base_plugins, disease, model, tmpd
     assert np.isclose(
         actual_person_times["with_condition"], expected_with_condition_person_time, rtol=0.001
     )
+
+
+def test_different_results_per_disease(base_config, base_plugins, tmpdir):
+    """Test that all eash disease observer saves out its own results."""
+    vampiris_healthy_state = SusceptibleState("not_a_vampire")
+    vampiris_infected_state = DiseaseState("a_vampire")
+    vampiris_healthy_state.add_rate_transition(vampiris_infected_state)
+    vampiris = DiseaseModel(
+        "vampiris",
+        initial_state=vampiris_healthy_state,
+        states=[vampiris_healthy_state, vampiris_infected_state],
+    )
+    hcd_healthy_state = SusceptibleState("not_a_zombie")
+    hcd_infected_state = DiseaseState("a_zombie")
+    hcd_healthy_state.add_rate_transition(hcd_infected_state)
+    human_cortico_deficiency = DiseaseModel(
+        "human_cortico_deficiency",
+        initial_state=hcd_healthy_state,
+        states=[hcd_healthy_state, hcd_infected_state],
+    )
+
+    vampiris_observer = DiseaseObserver("vampiris")
```

Comment:
Vampiris not vampirism?

Reply:
https://imlegend.fandom.com/wiki/Vampiris